### PR TITLE
OpenRefine Database Import Extension

### PR DESCRIPTION
**Pull request by tcbuzor**

### CODE REVIEW

File: extensions/database/module/index.vt [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#*
+ * Access this page at the URL /extension/database/
+ *#
+<html>
+  <head>
+    <title>Google Refine Database Extension v$version</title>
```
Comment (thadguidry) [inline extensions/database/module/index.vt]: Rename this with OpenRefine

File: extensions/database/module/index.vt [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  </head>
+  <body>
+    <h1>Google Refine DATABASE Extension v$version</h1>
```
Comment (thadguidry) [inline extensions/database/module/index.vt]: Rename this with OpenRefine

File: extensions/database/module/scripts/database-extension.js [hunk 3]
```diff
@@ -0,0 +1,206 @@
+/*
+
+Copyright 2010, Google Inc.
```
Comment (thadguidry) [inline extensions/database/module/scripts/database-extension.js]: uhh, you probably want to put your name here and not Google's ?  :)

File: extensions/database/module/scripts/index/database-import-controller.js [hunk 4]
```diff
@@ -0,0 +1,375 @@
+/*
+
+Copyright 2011, Google Inc.
```
Comment (thadguidry) [inline extensions/database/module/scripts/index/database-import-controller.js]: sames

File: extensions/database/module/scripts/project/database-exporters.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
+/*
+
+Copyright 2011, Google Inc.
```
Comment (thadguidry) [inline extensions/database/module/scripts/project/database-exporters.js]: sames

File: extensions/database/module/styles/database-import.less [hunk 6]
```diff
@@ -0,0 +1,152 @@
+/*
+
+Copyright 2011, Google Inc.
```
Comment (thadguidry) [inline extensions/database/module/styles/database-import.less]: sames

File: extensions/database/module/styles/theme.less [hunk 7]
```diff
@@ -0,0 +1,34 @@
+/*
+
+Copyright 2011, Google Inc.
```
Comment (thadguidry) [inline extensions/database/module/styles/theme.less]: sames

**Comment by wetneb** [comment]
@tcbuzor Many of the tests are empty - wouldn't it be better to provide actual test cases?

**Comment by tcbuzor** [comment]
Antonin,

I am almost done with the issues you raised. I will push the changes before
weekend.

-Tony

**Comment by wetneb** [comment]
@tcbuzor woaw, that's fantastic! so much testing! Thank you so much!

**Comment by thadguidry** [comment]
@tcbuzor @jackyq2015 Any idea why all of those failed "loading workspace" under /tmp on Travis ???  I wonder if there's a setting in Travis we need to enable/disable ?

```
[testng] 04:58:35.883 [       FileProjectManager] Loading workspace: /tmp/OR_DBExtension_Test_WorkspaceDir5488033601648434206/workspace.old.json (0ms)
   [testng] 04:58:35.883 [       FileProjectManager] Failed to load workspace from any attempted alternatives. (0ms)
   [testng] .04:58:35.929 [       FileProjectManager] Overwriting singleton already set: com.google.refine.io.FileProjectManager@217abcc9 (46ms)
```

**Comment by wetneb** [comment]
We need to disable these tests on Appveyor too, as Appveyor does not seem to support databases. Actually, we should disable the tests by default and only run them for Travis, so that folks don't have to launch three different database engines to test OpenRefine.

**Comment by thadguidry** [comment]
@wetneb stop lying. :)  https://www.appveyor.com/docs/services-databases/

**Comment by wetneb** [comment]
oh great! then why don't you configure appveyor so that it goes back to green? :-P

**Comment by thadguidry** [comment]
@wetneb because I'm retiring soon. :)

**Comment by wetneb** [comment]
@thadguidry I'm so rusted on windozy-things that I was hoping I could ignore the issue ^^

**Pull request merged**